src/modules/tokens/services/token.service.ts

```diff
@@ -44,6 +58,42 @@ export class TokenService {
         return tokens;
     }
 
+    async getFilteredTokens(
+        offset: number,
```

Comment:
Maybe for `offset` and `limit` you can create a new entity, same as QueryPagination in `api.multiversx.com`